internal/gen/jsonschema/buf.protoschema.test.v1.ConstraintTest.jsonschema.strict.bundle.json

```diff
@@ -0,0 +1,936 @@
+{
+  "$defs": {
+    "buf.protoschema.test.v1.ConstraintTest.RequiredImplicit.jsonschema.strict.json": {
+      "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
Is this required to be repeated in each definition or does it allow falling back to the `$schema` defined at the top-level?